tools/sigh.ts

```diff
@@ -422,6 +446,101 @@ function link(srcFiles: Iterable<string>): boolean {
   return success;
 }
 
+// Config for building wasm modules.
+type WasmConfig = {
```

Comment:
does it make sense for this to be an interface? When we use `type`, it creates an alias.

Reply:
Good idea; done.